eq-author/src/utils/withChangeUpdate.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+const withChangeUpdate = WrappedComponent => {
+  return class extends React.Component {
+    static propTypes = {
+      onChange: PropTypes.func.isRequired,
+      onUpdate: PropTypes.func.isRequired
+    };
+
+    handleUpdate = update => this.props.onChange(update, this.props.onUpdate);
```

Comment:
This looks remarkably to `withEntityTypeChange` is there any way we can reuse this?